src/components/Contentful/LandingPageHero.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+	<section-with-background :background-content="heroBackground">
+		<template #content>
+			<!-- same wrapper as src/components/BorrowerProfile/ContentContainer.vue -->
+			<kv-page-container>
+				<kv-grid class="tw-grid-cols-12">
+					<div class="tw-col-span-12 md:tw-col-start-2 md:tw-col-span-10 lg:tw-col-span-6">
+						<h1 v-html="heroHeadline">
+						</h1>
+						<h3 v-html="heroSubHeadline" class="tw-pt-2.5 lg:tw-pt-2">
```

Comment:
@ryan-ludwig any tailwinds best practices here on whether to use pt- class on the h3 or pb class on the h1 to add space between these 2 elements?

Reply:
I don't think there's a best practice per say. My usual system is to only use margin bottoms on everything to keep things predictable and avoid margin collapse weirdness.